src/Core/Components/AppBar/FluentAppBar.razor.cs

```diff
@@ -117,6 +131,11 @@ public async Task OverflowRaisedAsync(string value)
         await InvokeAsync(StateHasChanged);
     }
 
+    private async Task InitializeOverflowAsync()
+    {
+        await _jsModuleOverflow.InvokeVoidAsync("fluentOverflowInitialize", _dotNetHelper, Id, Orientation == Orientation.Horizontal, OVERFLOW_SELECTOR);
```

Comment:
You should test `_jsModuleOverflow` before executing this method.
It is likely that this object will not be null, but there is no guarantee that a Blazor event or unit test will try to execute it.